tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -672,6 +681,40 @@ func getNodeForReboot(isIPv6 bool) (string, string, error) {
 	return "", "", fmt.Errorf("no egress IP address found in egressIP map")
 }
 
+func setMinAvailableToZeroForActivePDB() error {
+	By("Workaround for the node drain failure due to active PodDisruptionBudget")
+
+	allAvailablePDB, err := poddisruptionbudget.ListInAllNamespaces(APIClient)
+
+	if err != nil {
+		glog.Infof("Failed to list all available pod disruption budget due to %v", err)
+
+		return nil
+	}
+
+	zeroInt := intstr.FromInt32(0)
```

Comment:
why not to use regular type conversion:
```
zeroInt := int32(0)
```

Reply:
because it is not suitable for the required variable type:Cannot use '&zeroInt' (type *int32) as the type *intstr. IntOrString